src/test/java/io/anserini/document/WikipediaArticleTest.java

```diff
@@ -36,7 +36,7 @@
 import io.anserini.collection.WikipediaCollection;
 
 
-public class WikipediaArticleTest extends DocumentTest {
+public class WikipediaArticleTest extends DocumentTest<TrecwebDocument> {
```

Comment:
WikipediaArticle is not a subclass of `TrecwebDocument`.
There must be something wrong with previous code (not a bug though)

Reply:
@Peilin-Yang So for wikipedia article, the `dType` should be `WikipediaArticle` instead of current `TrecwebDocument`, right?

Reply:
Yes.